src/lib.rs

```diff
@@ -0,0 +1,5 @@
+/* SPDX-License-Identifier: MIT */
+
+pub mod config;
+pub mod generator;
+pub mod setup;
```

Comment:
hmm... do we really need to make a library out of this?

Reply:
Yeah, I was wondering that too. Do you have a better way to allow `config` and `generator` to be imported?

Reply:
Apparently there is no better way when having tests in tests/ directory that would like to use library instead of calling to a binary. In future we might want to move the unit-tests to the src/ and have tests/ to just call a binary interface but I think it is good enough for now. Any testing is better than no testing :)